app/javascript/src/Users/types/index.jsx

```diff
@@ -0,0 +1,4 @@
+// @flow
+
+export * from 'Users/types/Role'
+export * from 'Users/types/Feature'
```

Comment:
Do we need 2 files `Feature.jsx`and `Role.jsx`? Maybe is too much, specially for a single-line like Role.jsx